### PR TITLE
style(settings): add gap to identity tags

### PR DESCRIPTION
**Pull request by michellewzhang**
before
SCR-20240828-keuh

after 
SCR-20240828-kerz

### CODE REVIEW

File: static/app/views/settings/account/accountIdentities.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ function IdentityItem({identity, onDisconnect}: IdentityItemProps) {
         </IdentityText>
       </InternalContainer>
       <InternalContainer>
-        <TagWrapper>
+        <TagWrapper css={{gap: space(0.75)}}>
```
Comment (evanpurkhiser) [inline static/app/views/settings/account/accountIdentities.tsx:67]: Why not add the property to the `TagWrapper` styled component?

**Comment by evanpurkhiser** [comment]
@michellewzhang small nit with your PR descriptions.

I would actually just prefer to have the before and after as two paragraphs. so the images expand to the full width of the comment. Makes it easier to see the difference without having to click them

File: static/app/views/settings/account/accountIdentities.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/views/settings/account/accountIdentities.tsx]: oops i totally missed that it's a styled component, just moved it

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Pull request merged**